doc/concepts/recipe.rst

```diff
@@ -8,29 +8,33 @@ building phase to provide information about cells in the model, such as:
 
   * The **number of cells** in the model.
   * The **kind** of each cell.
-  * The **description** of each cell, e.g. with morphology, synapses, detectors, stimuli.
+  * The **description** of each cell, e.g. with morphology, dynamics, synapses, detectors,
+    stimuli etc.
```

Comment:
The etc. throws me off again. Are the following bullets included for example? I think not but I'm not sure.

Reply:
A bit more context about the description: the cell description is basically the cell itself, which includes everything I listed plus other things I didn't: the initial voltage, the temperature, the initial ion concentrations etc. I think the confusion is around the `description` term. This is directly related to the `get_cell_description()` method of the recipe, which returns a cell object (in most cases, a `cable_cell`). Take for example the [branch_cell](https://github.com/arbor-sim/arbor/blob/master/example/ring/branch_cell.hpp#L56) function from our repo. It returns a full "description" of a cell, which is actually a `cable_cell` object. This function is called from [recipe::get_cell_description()](https://github.com/arbor-sim/arbor/blob/master/example/ring/ring.cpp#L80). 

Is this still vague? Should the line be: 
`The cells themselves, referred to as the cell description of each cell, e.g. with morphology, dynamics, synapses, detectors, stimuli etc.`

As for the etc, I can remove it, but I don't see the source of the confusion. I just mean to convey that there are things that can be included in the description of the cell other than morphology, dynamics, synapses and detectors.

Reply:
If the "cell description", as we use the term in Arbor, is not described here in full, then it isn't anywhere as far as I'm able to establish (which may be hindered by the fact 'description' is used in many contexts in the docs). I think this is the right place to do so.

`get_cell_description()` is documented in basically the same way and so does not offer further clarification.

Reply:
I am adding a `Cell descriptions` section in `concepts/cells` right underneath `Cell kinds`, as I think they are related to some extent. And I'll link to it from this page. Do you mind if this is in a separate PR? What changes do you suggest for this PR?

Reply:
OK, that's fine too. A link would be nice! Then go with 'etc.' for this PR.